File: Java/src/Manzano/Pagina100a.java
package Manzano;
//Ler  duas  matrizes  A  e  B,  cada  uma  de  duas  dimensões  com  5  linhas  e  3  colunas.  Construir  uma matriz  C  de  mesma  dimensão,  que  é  formada  pela  soma  dos  elementos  da  matriz  A  com  os elementos da matriz B. Apresentar a matriz C. 

import java.util.Scanner;

public class Pagina100a {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int[][] A = new int[5][3];
        int[][] B = new int[5][3];
        int[][] C = new int[5][3];

        for (int i = 0; i < 5; i++) {
            for (int j = 0; j < 3; j++) {
                System.out.println("Digite um número para ser inserido na matriz A: ");
                A[i][j] = scanner.nextInt();
                System.out.println("Digite um número para ser inserido na matriz B: ");
                B[i][j] = scanner.nextInt();
                C[i][j] = A[i][j] + B[i][j];
            }
        }
        for (int i = 0; i < 5; i++) {
            for (int j = 0; j < 3; j++) {
                System.out.print(C[i][j] + " ");
            }
            System.out.println();
        }

    }
}

File: JavaScript/Manzano/Pagina100a.js
// Ler duas matrizes A e B, cada uma de duas dimensões com 5 linhas e 3 colunas.
// Construir uma matriz C de mesma dimensão, que é formada pela soma dos elementos da matriz A com os elementos da matriz B.
// Apresentar a matriz C.

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const A = [];
const B = [];
const C = [];

function lerElementosMatrizes() {
  rl.question("Digite um número para ser inserido na matriz A: ", function (input) {
    const valorA = parseInt(input);
    rl.question("Digite um número para ser inserido na matriz B: ", function (input) {
      const valorB = parseInt(input);

      const linhaA = [];
      const linhaB = [];
      const linhaC = [];

      for (let i = 0; i < 3; i++) {
        linhaA[i] = valorA;
        linhaB[i] = valorB;
        linhaC[i] = valorA + valorB;
      }

      A.push(linhaA);
      B.push(linhaB);
      C.push(linhaC);

      if (A.length < 5) {
        lerElementosMatrizes();
      } else {
        apresentarMatrizC();
        rl.close();
      }
    });
  });
}

function apresentarMatrizC() {
  for (let i = 0; i < 5; i++) {
    for (let j = 0; j < 3; j++) {
      process.stdout.write(C[i][j] + " ");
    }
    process.stdout.write("\n");
  }
}

lerElementosMatrizes();